pyproject.toml

```diff
@@ -100,18 +100,7 @@ force_single_line = true
 
 [tool.pydocstyle]
 convention = "numpy"
-add_ignore = [
-    "D100",
-    "D101",
-    "D102",
-    "D103",
-    "D104",
-    "D105",
-    "D106",
-    "D107",
```

Comment:
just double checking, we want to have docstrings on classes AND their __init__ ?

Reply:
I would say so, yes. In general, init arguments and class attributes can differ. So, having docstrings for both makes sense in my opinion.